implementation/src/main/java/io/smallrye/config/Converters.java

```diff
@@ -173,6 +174,9 @@ private Converters() {
     static final Converter<Pattern> PATTERN_CONVERTER = BuiltInConverter.of(17,
             newTrimmingConverter(newEmptyValueConverter(Pattern::compile)));
 
+    static final Converter<Path> PATH_CONVERTER = BuiltInConverter.of(18,
+            newTrimmingConverter(newEmptyValueConverter(Path::of)));
```

Comment:
This one shouldn't be a trimming converter IMO. Paths may contain, start with, or end with spaces. Not that that's a good idea to do 🙂